### PR TITLE
better descriptions batch one

### PR DESCRIPTION
**Pull request by BeastFolks**
#### Summary

SUMMARY: Content "First batch of ten furniture JSONs that I've edited. More to come."

#### Purpose of change

I see loads of inconsistency in item and furniture descriptions, with wildly fluctuating tone, and either useful evocative descriptions, or completely useless token efforts at adding descriptions. I really wanted to contribute to CDDA, and I can't code for shit, so hopefully this helps.

#### Describe the solution

I've gone through a bunch of the descriptions for furniture items, specifically the first ten JSONs for furniture and terrain, and I've given them more descriptive and tonally consistent descriptions. I've tried to have them be straightforward and useful, without the weird out-of-place jokeyness in a lot of the descriptions.

#### Additional context

I'm going to be doing many more of these, this is just all I'm going to do for today.

### CODE REVIEW

**Comment by KorGgenT** [comment]
please use the correct summary line, found in the template

**Comment by RoyBerube** [comment]
Here is a link to regex test your summary line for correctness:
https://regex101.com/r/rZZTn0/2
Change the contents to match your PR. I find it useful.

**Comment by AlisW02** [comment]
> 
> 
> Here is a link to regex test your summary line for correctness:
> https://regex101.com/r/rZZTn0/2
> Change the contents to match your PR. I find it useful.

Thank you, I was having a hell of a time trying to figure that out.

File: data/json/furniture_and_terrain/furniture-medical.json [hunk 1]
```diff
@@ -492,7 +492,7 @@
     "type": "furniture",
     "id": "f_MS",
     "name": "mass spectrometer",
-    "description": "Inside this thing is a carefully balanced set of electric field generators that can precisely separate ionized particles based on their charge-to-mass ratio, firing them into a detector that measures the exact mass of the particle hitting it.  On the outside, it looks like a very boring white box.",
+    "description": "Inside this large white box is a carefully balanced set of electric field generators that can precisely separate ionized particles based on their charge-to-mass ratio, firing them into a detector that measures the exact mass of the particle hitting it.  Invaluable for particle research and other advanced sciences, it's not as useful anymore.",
```
Comment (Torbenbeta) [inline data/json/furniture_and_terrain/furniture-medical.json:495]: I am no expert for MS but in "normal" chemistry a MS is mostly used for analytical chemistry and not for particle research.
So I would change it to 
"Invaluable for analytical chemistry and other advanced sciences, it's not as useful anymore.","

File: data/json/furniture_and_terrain/furniture-medical.json [hunk 2]
```diff
@@ -533,7 +533,7 @@
     "type": "furniture",
     "id": "f_NMR",
     "name": "nuclear magnetic resonance spectrometer",
-    "description": "This is a giant electromagnet in a kind of sci-fi looking housing.  Somehow it can be used to wiggle molecular bonds or something, and from there, look at the deepest inner workings of chemical structures!  Magnets: how do they work?",
+    "description": "This is a giant electromagnet with carefully tuned measurement equipment used to observe how magnetic fields interact with nuclear particles.",
```
Comment (Torbenbeta) [inline data/json/furniture_and_terrain/furniture-medical.json:536]: I would change to :
"This is a giant electromagnet used to observe how  nuclear spins are affected by a magnetic field. The workhorse for the characterization of organic molecules." 
Alternatively: "[...] workhorse for the discovery of chemical structures"

File: data/json/furniture_and_terrain/furniture-medical.json [hunk 3]
```diff
@@ -573,7 +573,7 @@
     "type": "furniture",
     "id": "f_electron_microscope",
     "name": "electron microscope",
-    "description": "An enormous tool for using electron reflections off a surface to see what very tiny things look like.  Amazing for taking gross pictures of bugs.",
+    "description": "An enormous obersvational tool for studying the details of samples on an immensely small scale.",
```
Comment (Torbenbeta) [inline data/json/furniture_and_terrain/furniture-medical.json:576]: Just a typo: observational.

Comment (AlisW02) [reply data/json/furniture_and_terrain/furniture-medical.json:576]: Oh thank _christ,_ I was really hoping somebody who knows what this stuff means haha, can you tell that I flunked chemistry? Anyway, jokes aside, thank you very much, I'll make these revisions.

File: data/json/furniture_and_terrain/furniture-medical.json [hunk 4]
```diff
@@ -402,7 +402,7 @@
     "type": "furniture",
     "id": "f_HPLC",
     "name": "high performance liquid chromatographer",
-    "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a liquid or aqueous state, based on their affinity, into a solid state in a test tube.  At least, that's what the label says.",
+    "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a liquid or aqueous phase, based on their affinity, to a solid state in a tube.  At least, that's what the label says.",
```
Comment (Torbenbeta) [inline data/json/furniture_and_terrain/furniture-medical.json:405]: After some more thinking ¨based on their affinity, to the stationary phase in the tube." seems to me even better and 
scientifically more correct.

File: data/json/furniture_and_terrain/furniture-medical.json [hunk 5]
```diff
@@ -447,7 +447,7 @@
     "type": "furniture",
     "id": "f_GC",
     "name": "gas chromatographer",
-    "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a gaseous state, based on their affinity, to a solid state in a test tube.  At least, that's what the label says.",
+    "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a gaseous phase, based on their affinity, to a solid state in a tube.  At least, that's what the label says.",
```
Comment (Torbenbeta) [inline data/json/furniture_and_terrain/furniture-medical.json:450]: Same as with LC.

**Comment by AlisW02** [comment]
> 
> 
> Thank you for incorporating my suggestions. :-)

Sure thing! I've just added those two other changes suggested. Just let me know if you see anything else!

**Comment by kevingranade** [comment]
Do keep an eye on the tests, there are some style checks that you can trigger (and you did) that the tests will catch for you.
https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/687544646

**Comment by AlisW02** [comment]
> 
> 
> Do keep an eye on the tests, there are some style checks that you can trigger (and you did) that the tests will catch for you.
> https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/687544646

I'm afraid I can't find exactly what the error was, when I checked the link, everything said well-formatted for the stuff I modified, unless I missed something?

**Comment by anothersimulacrum** [comment]
https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/687544646#L4264

**Comment by AlisW02** [comment]
> 
> 
> https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/687544646#L4264

THERE we go, thank you! Gods, I knew those spaces would get me in the end

**Comment by AlisW02** [comment]
Okay, so, I need some more help. I've been trying to find the typos the error log mentioned, but I couldn't find them. I might just be way too tired for this, but I can't help but worry that there's something more going on.

File: data/json/furniture_and_terrain/furniture-fungal.json [hunk 6]
```diff
@@ -3,7 +3,7 @@
     "type": "furniture",
     "id": "f_flower_marloss",
     "name": "marloss flower",
-    "description": "This flower is like the other flowers taken by the mushrooms, but its bulb is colored a brilliant cyan color, and it emits an aroma both overwhelming and… delicious?",
+    "description": "This flower is like the other flowers taken by the mushrooms, but its bulb is colored a brilliant cyan color. It emits an aroma both overwhelming and strangely alluring.",
```
Comment (nphyx) [inline data/json/furniture_and_terrain/furniture-fungal.json:6]:
```suggestion
    "description": "This flower is like the other flowers taken by the mushrooms, but its bulb is colored a brilliant cyan color.  It emits an aroma both overwhelming and strangely alluring.",
```
This was the only remaining complaint in the error log that I could find.

**Comment by AlisW02** [comment]
AHA, I KNEW THERE WAS AN ACCIDENTAL SINGLE SPACE SOMEWHERE, THANK YOU

**Pull request merged**